setup.py

```diff
@@ -47,8 +47,8 @@
     extras_require={
         'dev': ['black', 'isort', 'pytest<8.0', 'flake8'],
         'atari': ['gym[atari, accept-rom-license]'],
-        'mujoco': ['gym[mujoco]<1.0', 'mujoco', 'mujoco_py>=2.0,<3.0'],
-        'vizdoom': ['vizdoom<2.0'],
+        'mujoco': ['gym[mujoco, mujoco_py]'],
+        'vizdoom': ['vizdoom<2.0', 'gym[classic_control]'],
```

Comment:
I'm assuming this is for pygame dependency? Cool.

Reply:
Yes, it is for pygame